### PR TITLE
Building rpm package with docker on CircleCI

### PR DESCRIPTION
**Pull request by pastak**
ref: #23

### CODE REVIEW

**Comment by pastak** [comment]
This is forked PR #23 from @imkitchen .

It builds rpm package on CircleCI.

**Comment by pastak** [comment]
@drobune @gyazo/owners Please review it

File: rpm_docker.sh [hunk 1]
```diff
@@ -0,0 +1,7 @@
+#!/bin/bash
+
+sudo service docker start
+docker build -t centos:latest .
+docker cp `docker ps -l -q`:/root/rpmbuild/RPMS/x86_64/${PACKAGE}-${VERSION}-1.x86_64.rpm ./
```
Comment (drobune) [inline rpm_docker.sh]: Could run `package_cloud push ~~`  in docker??
It is simple that rpm build and push by one liner `docker run` , I think.

Comment (pastak) [inline rpm_docker.sh]: When `package_cloud` runs on CentOS container, gem requires `LANG UTF-8`.

I set ENV `LANG=en_US.UTF8` but same message appered and failed... :crying_cat_face: 

Do you know any ideas? Please help

Comment (imkitchen) [inline rpm_docker.sh]: I have confirmed the error in my environment that @pastak has reported.
```bash
% docker build -t centos:latest .
...
It appears your locale is set to US-ASCII, please use UTF-8 instead. Run the 'locale' command to getmore info or e-mail support@packagecloud.io for help
INFO[0283] The command [/bin/sh -c package_cloud push imkitchen/gyazo/el/7 ./gyazo*.rpm] returned a non-zero code: 1 
```

In Dockerfile, you can use ``ENV`` statement as follow.
```dockerfile
# change locale to en_US.utf8
ENV LANG en_US.utf8
ENV LANGUAGE en_US:en
ENV LC_ALL en_US.utf8
```

Comment (pastak) [inline rpm_docker.sh]: I'll try it :rocket:

**Comment by pastak** [comment]
@drobune I succeed `packagecloud push ~~` by `docker run`. thanks @imkitchen comment :musical_note:

**Comment by pastak** [comment]
Please someone check you can install via

```
$ curl -s https://packagecloud.io/install/repositories/gyazo/gyazo-for-linux/script.rpm.sh | sudo bash
$ sudo yum install gyazo
```

**Comment by drobune** [comment]
thx guys!!! :sunglasses: 

LGTM

[Gyazo](https://gyazo.com/cbd972c7a4e7a3a63431af1d73320ba5)

**Pull request merged**